python/QC3.py

```diff
@@ -0,0 +1,28 @@
+import datetime,os,sys
+import subprocess
+from subprocess import CalledProcessError
+from wrappers import runCommand
+from options import parser
+
+
+#print args.file
+
+cmd1 = ["python","/Users/anastasia/desktop/QC_Plotting_Tools/Produce_Config_File.py"]
```

Comment:
There's only one command list here `cmd1`, why does it get a number? Change.

Hard coded path, change to get the path from the OS, for example:

https://github.com/bdorney/gem-plotting-tools/blob/master/ana_scans.py#L20

The setup script of the repository sets the `$SHELL` variable `$GEM_BASE.`  Use `$GEM_BASE` to ensure user/system independent operation.